test/specgen/route-helper.test.js

```diff
@@ -475,6 +475,17 @@ describe('route-helper', function() {
     expect(doc.operation.responses).to.not.have.property(204);
   });
 
+  it('does not include `undefined` error response in `responseMessages` when ' +
+    'http is set, but erorrStatus is not set', function() {
+    const doc = createAPIDoc({
+      http: {
+        status: 201,
+      },
+    });
+
+    expect(Object.keys(doc.operation.responses)).to.have.length(1);
```

Comment:
When the assertion `have.length(1)` fails, it produces error message that's usually not very helpful (expected 2 to be 1). Let's verify the exact items in the array instead.

```suggestion
    expect(Object.keys(doc.operation.responses)).to.eql([201]);
```